Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -180,3 +180,29 @@ def customiseRun2CommonHI(process):
 
     return process
 
+##############################################################################
+# modifications of muon DQMOffline for pPb run
+def customiseRun2PPB_MuonDQMOffline(process):
```

Comment:
DQMOffline/Configuration
that's where I see similar mods in DQMOfflineHeavyIons_cff.py